histomicstk/features/FeatureExtraction.py

```diff
@@ -1,12 +1,15 @@
-import collections
-import math
 import numpy as np
 import pandas as pd
-from scipy.stats import entropy, skew, kurtosis
 from skimage.feature import canny
 from skimage.measure import regionprops
-from skimage.segmentation import find_boundaries
-from skimage.morphology import disk, dilation
+from skimage.morphology import disk, binary_dilation
+
+from .ComputeFSDFeatures import ComputeFSDFeatures
+from .ComputeGradientFeatures import ComputeGradientFeatures
+from .ComputeIntensityFeatures import ComputeIntensityFeatures
+from .ComputeMorphometryFeatures import ComputeMorphometryFeatures
```

Comment:
True - area alone is not shape. Neither is perimeter. But together they provide shape cues. MajorAxisCoordsX/Y also just indicate orientation, not shape.

I don't think there are enough size features that exist to create a separate category, and shape seems to be the closest category (containing features that depend only on the binary mask of an object).